openmmtools/tests/test_integrators.py

```diff
@@ -429,7 +429,7 @@ def test_thermostated_integrator_hash():
     assert len(all_hashes) == len(thermostated_integrators)
 
 
-def test_alchemical_langevin_integrator():
+def run_alchemical_langevin_integrator(nsteps=0):
```

Comment:
well, another function runs and calls it, so that's right, it shouldn't be run directly.